src/App.tsx

```diff
@@ -1,25 +1,45 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
 
 const App: React.FC = () => {
   const [
     selectedUserId,
-    // setSelectedUserId,
+    setSelectedUserId,
   ] = useState(0);
 
+  const [todos, setTodos] = useState([]);
+
+  useEffect(() => {
+    getTodos().then(result => {
+      setTodos(result);
+    });
+  }, []);
+
+  const changeUser = (id: number): void => {
+    setSelectedUserId(id);
+  };
+
   return (
     <div className="App">
       <div className="App__sidebar">
-        <TodoList />
+        <TodoList
```

Comment:
TodoList should be responsible solely for rendering a list of todos
pass into TodoList visible todos, move all inputs here, into an App component